<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Демонстрация мандатной модели</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>0</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_dirs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Секретность</string>
    </property>
    <widget class="QRadioButton" name="radioButton_TS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>TopSecret</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_S">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Secret</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_N">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>NoSecret</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ws_path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_choose_workdir">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>50</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>выбор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_update">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить данные</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>392</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_dirs_from">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_dirs_to">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбор рабочей директории:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;From&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;To&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_files">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>351</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_copy">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>280</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Копировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>220</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выбор файла&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
